distribution/src/bin/elasticsearch.bat

```diff
@@ -54,7 +90,12 @@ if "%MAYBE_JVM_OPTIONS_PARSER_FAILED%" == "jvm_options_parser_failed" (
   exit /b 1
 )
 
-%JAVA% %ES_JAVA_OPTS% -Delasticsearch -Des.path.home="%ES_HOME%" -Des.path.conf="%ES_PATH_CONF%" -Des.distribution.flavor="%ES_DISTRIBUTION_FLAVOR%" -Des.distribution.type="%ES_DISTRIBUTION_TYPE%" -Des.bundled_jdk="%ES_BUNDLED_JDK%" -cp "%ES_CLASSPATH%" "org.elasticsearch.bootstrap.Elasticsearch" !newparams!
+ECHO.%KEYSTORE_PASSWORD%| %JAVA% %ES_JAVA_OPTS% -Delasticsearch ^
```

Comment:
Yikes - just realized that this needs to be a delayed expansion of `KEYSTORE_PASSWORD` or else special characters in passwords will be parsed. More dangerously, someone could enter malicious instructions as an incorrect password. See: https://www.robvanderwoude.com/battech_inputvalidation_setp.php#SetP